docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.3"
+
+services:
+  database:
+    image: mysql
+    restart: on-failure
+    ports:
+      - 3306:3306
+    expose:
+      - 3306
+    environment:
+      - MYSQL_DATABASE=hollowverse
+      - MYSQL_ROOT_PASSWORD=123456
+      - MYSQL_ROOT_HOST=%
+
+  api:
+    restart: always
```

Comment:
What's `always`?

Reply:
`always` means that the container will restart whenever the entry point command (`yarn dev`) exits, regardless of the exit code. `yarn dev` is not supposed to exit at all so I'm not sure about using `always`. `on-failure` could also work.